django_celery_extensions/management/commands/run_celery.py

```diff
@@ -1,33 +1,34 @@
 import os
 
 import shlex
-import subprocess
 
 from django.conf import settings as django_settings
 from django.core.management.base import BaseCommand
 from django.utils.autoreload import run_with_reloader, DJANGO_AUTORELOAD_ENV
 
-import signal
-
-
-def kill_celery(celery_type, celery_settings):
-    kill_celery_command = f'pkill -9 -f "{get_celery_command(celery_type, celery_settings, add_queue=False)}"'
-    subprocess.call(shlex.split(kill_celery_command))
+from celery.bin.celery import celery
 
 
 def autoreload_celery(celery_type, celery_settings, extra_arguments):
-    kill_celery(celery_type, celery_settings)
-    subprocess.call(shlex.split(get_celery_command(celery_type, celery_settings, extra_arguments)))
+    celery(shlex.split(get_celery_command(
+        celery_type=celery_type,
+        celery_pool='solo',
```

Comment:
in the dev mode celery is run only in solo mode (only one thread) this will simplify restarting (no kill command for subprocess)

Reply:
this has one little problem. Solo ignore task time limits. So If you want to test it, you must run celery without autoreload